src/Handler/MockHandler.php

```diff
@@ -61,8 +61,9 @@ public static function createWithMiddleware(
      * {@see \Psr\Http\Message\ResponseInterface} objects, Exceptions,
      * callables, or Promises.
      *
-     * @param callable|null $onFulfilled Callback to invoke when the return value is fulfilled.
-     * @param callable|null $onRejected  Callback to invoke when the return value is rejected.
+     * @param array<int, mixed> $queue       The parameters to be passed to the append function, as an indexed array.
```

Comment:
What's the benefit of the typehint of queue here? even if the typehint here stays it needs to be `|null`

Reply:
This defines that `$queue` is an array with `mixed` content, which is an improvement over no typehint.

Added `|null` in ce607dbd0c5df30f85cea13072772b133f7b78dc

In addition I've added descriptive phpdoc to what it's used for, which is copied from the `call_user_func_array` docs.

> The parameters to be passed to the append function, as an indexed array.